Libplanet.Benchmarks/DataModel/DataModelBenchmark.LeafModel.cs

```diff
@@ -0,0 +1,91 @@
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using Libplanet.Crypto;
+using Libplanet.Store;
+using BTypes = Bencodex.Types;
+
+namespace Libplanet.Benchmarks
```

Comment:
```suggestion
namespace Libplanet.Benchmarks.DataModel
```

I think it'd be better to follow the directory structure.

Reply:
OK💯